test/TestParse.flix

```diff
@@ -51,4 +51,42 @@ mod TestParse {
     def testEscape01(): Bool = {
         parse("{\"key\": \"\\a\"}") == None
     }
+
+    def toIntOpt(e:Json.JsonElement): Option[Int32] = match e {
+        case JsonNumber(b) => BigDecimal.tryToInt32(b)
+        case _ => None
+    }
+
+    def parseAndSum(str:String, len:Int32, result:Int32): Bool =
```

Comment:
Changed the test for long array to calculate expected sum. Not sure what exactly you had in mind, feel free to suggest more details.

Reply:
I meant that instead of getting the sum, we can just create the entire expected value. For example, the `testShortArray` function would instead be written `parse("[1, 2, 3]") == Some(JsonArray(1ii :: 2ii :: 3ii :: Nil))`. Similarly for the long array, we could construct the full list and compare exactly.

Reply:
If you fix that, I'll merge and release a patch.

Reply:
Ah, I see! Changed it now.